staging/src/k8s.io/apiserver/pkg/audit/metrics.go

```diff
@@ -81,7 +81,7 @@ func HandlePluginError(plugin string, err error, impacted ...*auditinternal.Even
 	msg := fmt.Sprintf("Error in audit plugin '%s' affecting %d audit events: %v\nImpacted events:\n",
 		plugin, len(impacted), err)
 	for _, ev := range impacted {
-		msg = msg + EventString(ev) + "\n"
+		msg = msg + EventString(ev)
```

Comment:
Thanks for you review.
I didn't say clearly in commit message.
This change remove the empty lines between each event.
Before this change:
http://paste.openstack.org/show/611449/
After this change:
http://paste.openstack.org/show/611450/

We can see that some empty lines are removed.